Update changelog for v5.0 release (#3333)
mobilecoinfoundation/mobilecoin: CHANGELOG.md

## [5.0.0]

### Added

> - Average slam rate to json slam report ([#3080])

Reviewer: Can you change `Average` to some verb that describes the change to the slam rate?

Author: reworded

- Provide build time error messages for invalid SGX_MODE and IAS_MODE environment variables ([#3164])

> - `mc-attest-verifier-config` crate to handle multiple enclave measurements and hardening advisories ([#3148])

Reviewer: ```suggestion
- Use `mc-attest-verifier-config` crate to handle multiple enclave measurements and hardening advisories ([#3148])
```

Author: reworded

> - Logic in consensus to track and drop clients that exceed a number of failed tx proposals ([#3208], [#3155], [#3156])

Reviewer: ```suggestion
- Add logic in consensus to track and drop clients that exceed a number of failed tx proposals ([#3208], [#3155], [#3156])
```

Author: reworded

> - Passing extra GRPC services to `AdminServer` ([#3242])

Reviewer: ```suggestion
- Pass extra GRPC services to `AdminServer` ([#3242])
```

Author: reworded

> - [MCIP 61]: Defragmentation Memos ([#3170])

Reviewer: ```suggestion
- [MCIP 61]: Add Defragmentation Memos ([#3170])
```

Author: reworded

> - Capability of b58-decoder to print public address hashes ([#3283])

Reviewer: ```suggestion
- Enable b58-decoder to print public address hashes ([#3283])
```

Author: reworded

> - Fog view router to support sharding the fog storage. Provides a streaming and unary GRPC api. ([#3297])

Reviewer: ```suggestion
- Fog view router supports sharding the fog storage. Provides a streaming and unary GRPC api. ([#3297])
```

Author: reworded

> - Fog ledger router to support sharding the key image storage. Provides a streaming and unary GRPC api. ([#3312], ([#3331])

Reviewer: ```suggestion
- Fog ledger router supports sharding the key image storage. Provides a streaming and unary GRPC api. ([#3312], ([#3331])
```

Author: reworded
